index.js

```diff
@@ -337,7 +389,12 @@ module.exports = function (schema, options) {
                 deletedId: true,
             }
         };
+        
+        const saveOptions = {};
+        if (params.session) {
+            saveOptions.session = params.session;
+        }
 
```

Comment:
🎗️ Same question here about the `options` check, is it because it's a static that you don't need it?
```suggestion
        if(options.validateBeforeDelete === false) {
            saveOptions.validateBeforeSave = false;
        }
```